tests/python/gpu/test_operator_gpu.py

```diff
@@ -745,7 +745,7 @@ def test_pooling_with_type():
 @with_seed()
 def test_pooling_versions():
     def test_pooling_versions_helper(pool_op_list, data, kernel, pool_type, pad, stride,
-                                     pooling_convention='valid', global_pool=False):
+                                     pooling_convention='valid', global_pool=False, p_value=2):
```

Comment:
why default to 2?

Reply:
The most commonly used case for Lp pooling is p=2, for all other types of pooling this value is not used at all, so there's no side effect on other types of pooling with setting p_value to be 2.